src/common/system_logger.cc

```diff
@@ -0,0 +1,28 @@
+#include "src/common/system_logger.h"
+
+namespace gfs {
+
+namespace common {
+
+// TODO(bmokutub): If needed, also configure minloglevel to filter out certain
+// log levels. And log to files too.
+void SystemLogger::Initialize(const std::string& program_name) {
+  if (this->is_initialized_) {
+    // Already initialized. Should be done once.
+    return;
+  }
+
+  // Log to console
+  FLAGS_logtostderr = true;
```

Comment:
Could you please also show an example, at least  in comment regarding how to log to a file?

Reply:
I think Michael mentioned right now we only support log to console in first iteration

Reply:
Just logging to console right now. And you don't need to do anything from the log user perspective to log to file. Just use the macros the normal way. I will add the configuration for file logging in the system_logger initialization code. So will also log to file. I actually already have code for that, it's like two lines, but was seeing some weird behavior on my machine and didn't have time to debug it yet, so wanted to get this in first. And will add that after. Will also add comment for this question too. Good one.

Reply:
I also have couple of comments saying just logging to console right now.